lib/token.proto

```diff
@@ -3,13 +3,35 @@ syntax = "proto3";
 package hwsc;
 
 enum Permission {
-    ADMIN = 0;
+    NO_PERM = 0;
```

Comment:
This used to be an `ADMIN`. It turns GoLang will use a zero value or ADMIN permission level thus the change.